src/Experimental.cmake

```diff
@@ -106,9 +106,6 @@ set( EXPERIMENTAL_OPTIONS_LIST
    # PRL disabled 31 Aug 2021 because, at least on Mac, it misfires the
    # preference dialog whenever any keystroke shortcuts with Shift+ are used
    # EASY_CHANGE_KEY_BINDINGS
-
-   # Poke 13 Nov 2019
-   SUBRIP_LABEL_FORMATS
```

Comment:
This one actually only got merged this friday; somehow this feature had fallen through the cracks all these years. It is somewhat unfinished (see https://github.com/audacity/audacity/issues/6186) still and possibly the only flag I've seen so far which actually warrants an EXPERIMENTAL flag - but then, I did decide that in the current form it is useful and have no intention on disabling it if anything goes wrong. Removing this flag is fine.